comps/lvms_pg/lvm.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import json
+import os
+import time
+
+import requests
+from predictionguard import PredictionGuard
+
+from comps import (
+    LVMDoc,
+    ServiceType,
+    TextDoc,
+    opea_microservices,
+    register_microservice,
+    register_statistics,
+    statistics_dict,
+)
+
+
+@register_microservice(
+    name="opea_service@lvm_pg",
+    service_type=ServiceType.LVM,
+    endpoint="/v1/lvm",
+    host="0.0.0.0",
+    port=8091,
+    input_datatype=LVMDoc,
+    output_datatype=TextDoc,
+)
+@register_statistics(names=["opea_service@lvm_pg"])
+async def lvm(request: LVMDoc):
+    api_key = os.getenv("PREDICTIONGUARD_API_KEY")
```

Comment:
I think if `PREDICTIONGUARD_API_KEY` is an env var, you don't have the explicitly do the `api_key=api_key`.

Reply:
fixed